tests/raftstore_cases/test_prevote.rs

```diff
@@ -91,6 +93,9 @@ fn test_prevote<T: Simulator>(
         "Sends a PreVote or PreVoteResponse during failure.",
     );
 
+    // Allow the cluster to settle down.
+    thread::sleep(election_timeout * 2);
```

Comment:
I think we only needs to sleep one heartbeat interval, because there may be some prevote requests before leader's heartbeat arrival.

Reply:
Ok I will try this. :+1:

Reply:
It works. :sunrise: